src/pages/account/AdminPage.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import styled from "styled-components";
+
+import axiosApi from "../../utils/AxiosApi";
+import Card from "../../components/Card";
+import NavBar from "../../components/NavBar";
+import Button from "../../components/Button";
+
+const AdminPage = () => {
+  const [stories, setStories] = useState([]);
+  const [child, setChild] = useState({});
+  const [id, setId] = useState("");
```

Comment:
`id` for a `useState` variable name is not a good name. Id's are usually something generic as it's a term that is used on a lot of places. So be more specific what kind of id you're stating here